_includes/head.html

```diff
@@ -62,7 +62,6 @@
   <script>
   // Default to assuming this is an archive and hiding some stuff
   // See js/archive.js and js/docs.js for logic relating to this
-  var isArchive = true;
   var dockerVersion = 'v{{ site.docker_ce_version }}';
```

Comment:
not for this PR, but this one kept confusing me (we're setting it both here, and in the `_body.tml`